MolokwuChidera/EventMan
# Add .gitignore to ignore node_modules

#### What does this PR do? 
It creates API endpoints 

#### Description of Task to be completed
Features to create center and events were added

#### How should this be manually tested
It should be tested using post man by making a request to localhost:3000/centers for the centers page
and localhost:3000/events for the events page

####What are the relevant tracker stories
Setup and testing of API endpoints that can create center and events with dummy / in-memory data:
